.travis.yml

```diff
@@ -0,0 +1,113 @@
+language: php
+
+sudo: false
+
+dist: trusty
+
+addons:
+  postgresql: "9.5"
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - $HOME/.npm
+
+php:
+  - 7.1
+  - 7.2
+
+services:
+  - mysql
+
+env:
+  - DB=pgsql MOODLE_BRANCH=MOODLE_32_STABLE CI_PLUGIN=2
```

Comment:
We don't need a full matrix, we just need enough to know how deep support it. We want to have travis finish as soon as possible, like < 20 mins

I'd trim this back to the oldest know version, and then all the LTS versions, and then ones still in support

So I think that means culling 3.2, 3,3. 3.4